Resources/templates/CommonAdmin/object_actions.php.twig

```diff
@@ -35,17 +35,23 @@
 
     {% spaceless %}
         {{ echo_spaceless() }}
+        <div {% if builder.yamlKey is same as('list') or builder.yamlKey is same as('nested_list') %}
```

Comment:
Are you sure this new "parent" doesn't break buttons display?
Given that object actions are wrapped by this code:

```twig
<div class="col-md-12 btn-group">
    {{ block('object_actions') }}
</div>
```

I'm wondering if:

```html
<div class="col-md-12 btn-group">
    <div>
        <a class="btn btn-default">button1</a>
        <a class="btn btn-default">button2</a>
        <a class="btn btn-default">button3</a>
    <div>
</div>
```

Would be rendered the same as:

```html
<div class="col-md-12 btn-group">
    <a class="btn btn-default">button1</a>
    <a class="btn btn-default">button2</a>
    <a class="btn btn-default">button3</a>
</div>
```

(Just a question, I don't remember how BS handle `btn-group`)